src/containers/About/About.js

```diff
@@ -0,0 +1,182 @@
+/*
+Copyright 2020 The Tekton Authors
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import { injectIntl } from 'react-intl';
+import { Table } from '@tektoncd/dashboard-components';
+import { InlineNotification } from 'carbon-components-react';
+import { getInstallProperties } from '../../api';
+import { isWebSocketConnected } from '../../reducers';
+
+const initialState = {
+  dashboardInfo: [],
```

Comment:
this is being used as an object later, so we should be consistent and use an empty object for the initialState